cirq-core/cirq/protocols/json_test_data/XPowGate3Dim.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Instead of creating a new json file `XPowGate3Dim`, you should simply update the `XPowGate.json` and corresponding `repr` files. The same is true for `ZPowGate` as well. 

So, replace the current contents of `XPowGate.json` with 

```
[
  {
    "cirq_type": "XPowGate",
    "exponent": 0.123,
    "global_shift": 0.0
  },
  {
    "cirq_type": "XPowGate",
    "exponent": 0.123,
    "global_shift": 0.0,
    "dimension": 3
  }
]
```

the replace the current contents of XPowGate.repr with
```
[(cirq.X**0.123), cirq.XPowGate(exponent=0.123, dimension=3)]
```

Reply:
I see! I went up from `XPowGate.json` looking for examples of multiple in one test, but should have gone down to `XXPowGate.json` Oh well! I think that should be fixed now.